metacat-main/src/main/java/com/netflix/metacat/main/manager/ConnectorManager.java

```diff
@@ -41,26 +42,39 @@
 import com.netflix.metacat.common.server.connectors.exception.CatalogNotFoundException;
 import com.netflix.metacat.common.server.properties.Config;
 import com.netflix.metacat.common.server.spi.MetacatCatalogConfig;
+import lombok.AllArgsConstructor;
+import lombok.Data;
 import lombok.extern.slf4j.Slf4j;
 
 import javax.annotation.Nonnull;
 import javax.annotation.PreDestroy;
+import java.util.List;
 import java.util.Map;
+import java.util.Set;
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.ConcurrentMap;
 import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.stream.Collectors;
 
 /**
  * Connector manager.
  */
 @Slf4j
 public class ConnectorManager {
+    private static final String EMPTY_STRING = "";
```

Comment:
Actually. this (ConnectorManager) looks more like a catalog manager and the catalogManger is more like a connectorManager. We should switch their name.

Reply:
CatalogManager loads catalogs. ConnectorManager provides the right connector properties/services.